src/components/TimeseriesSearch/TimeseriesSearch.tsx

```diff
@@ -69,9 +74,11 @@ export class TimeseriesSearch extends React.Component<
 > {
   static defaultProps = {
     selectedTimeseries: [],
+    strings: {},
     filterRule: undefined,
     hideSelected: false,
     allowStrings: false,
+    hideRootAssetSelect: false,
```

Comment:
Maybe better to have the same prop value as for Search component (set to `false` by default)?
```typescript tsx
<TimeseriesSearch 
  ...
  rootAssetSelect={true}
/>
```